extensions/stripe/src/utils/cards.ts

```diff
@@ -0,0 +1,42 @@
+import type { Card } from "../data/cards";
+
+const createCardKey = (card: Card) => {
+  const key = `${card.name}:${card.number}`;
+  return key;
+};
+
+const createExpiryDate = () => {
+  const now = new Date();
+
+  const month = now.getMonth();
+  const year = String(now.getFullYear() + 1).slice(-2);
+
+  const expiryDate = `${month}/${year}`;
+  return expiryDate;
+};
+
+const createAppleScript = (card: Card) => {
```

Comment:
**thought:**


This works great while Raycast is consistently used in the MacOS ecosystem, this might be something we need to revisit down the line when [Raycast comes to Windows](https://www.raycast.com/windows). We can go with this approach for now.

Reply:
We should absolutely consider this. I would assume that in the future Raycast will implement a similar utility function like [runAppleScript](https://developers.raycast.com/utilities/functions/runapplescript) on Windows.